modules/mobile/spec/request/payment_history_request_spec.rb

```diff
@@ -48,11 +48,36 @@
           }
         )
       end
+
+      it 'returns meta data that includes the years in which the user has payments' do
+        expect(response.parsed_body['meta']).to include(
+          {
+            'availableYears' => [2019, 2018, 2017, 2016, 2015]
+          }
+        )
+      end
+
+      it 'only paginates and returns the number of records for the latest year' do
+        expect(response.parsed_body['meta']).to include(
```

Comment:
I think this test would be more valuable if we did something like map all the ids or dates from the returned payment data and confirm that they're the correct ones. Otherwise, we're just going by "i expect 7 entries and i got 7" but there's nothing to indicate that they're the correct 7. The same comment applies below.